denoise.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+import numpy as np
+import wave
+import math
+
+def nextpow2(n):
+    return np.ceil(np.log2(np.abs(n))).astype('long')
+
+
+
+f = wave.open("denoise/output.wav")
+# (nchannels, sampwidth, framerate, nframes, comptype, compname)
+params = f.getparams()
+nchannels, sampwidth, framerate, nframes = params[:4]
+fs = framerate
+str_data = f.readframes(nframes)
+f.close()
+x = np.fromstring(str_data, dtype=np.short)
+len_ = 20 * fs // 1000 
+PERC = 50 
+len1 = len_ * PERC // 100  

```

Comment:
`PERC` only used once  in line 21 too.

Reply:
Sorry, that's really my bad habbits

Reply:
Thats ok, if you think it has a purpose we can keep it but I'm not sure what `PERC` means so its up to you.